docs/website/docs/plus/ecosystem/snowflake_plus.md

```diff
@@ -7,13 +7,14 @@ keywords: [Snowflake, Iceberg, destination]
 # Snowflake Plus
 
 Snowflake Plus is an **experimental** extension of the [Snowflake destination](../../dlt-ecosystem/destinations/snowflake.md) that adds [Apache Iceberg tables](https://docs.snowflake.com/en/user-guide/tables-iceberg) creation and related features.
-This destination is available starting from `dlt-plus==0.9.0`. It fully supports all the functionality of the standard Snowflake destination, plus:
+This destination is available starting from `dlt-plus` version `0.9.0`. It fully supports all the functionality of the standard Snowflake destination, plus:
```

Comment:
is there a reason we're not just using dlt+ instead of `dlt-plus`?

Reply:
I was referring the package name, but I'm open to rephrasing this, since I mention the package name in installation anyways.

Reply:
Updated in 331ca45